tools/cli/pkg/ers/fetcher/initial.go

```diff
@@ -34,6 +34,9 @@ func (e InitialFetcher) GetAllInstances() ([]ers.Instance, error) {
 
 		output = append(output, instances...)
 		instances, err = e.client.GetPaged(page, e.pageSize)
+		if err != nil {
+			break
```

Comment:
maybe log the error at least? or aggregate and return all at the end?

https://github.com/kubernetes/apimachinery/blob/d407afb45eb78bce52710653e5f6c99e0d3493a9/pkg/util/errors/errors.go#L46